nginx.conf

```diff
@@ -17,7 +17,7 @@ http {
         error_page 400 403 404 /404;
         gzip on;
         gzip_types application/javascript application/json image/svg+xml text/css
-            text/plain;
+            text/plain application/font-ttf;
```

Comment:
One more thing. The MIME type should be `font/ttf`.

Reply:
i guess whichever tutorial i got this from is outdated haha